ebsco_adapter/ebsco_adapter/src/compare_uploads.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+
+def find_uploads_to_compare(available_files, xml_s3_prefix, s3_store):
+    available_files_dates = list(available_files.keys())
+    dates_list = sorted(available_files_dates, reverse=True)
+
+    print(f"Available uploads: {dates_list}")
+
+    # Check if we've sent a notification for the date
+    notified_completion_flag = "notified.flag"
+    date_list_with_notified_flag = []
+    for date in dates_list:
+        notified_completion_flag_path = os.path.join(
```

Comment:
I'd be tempted to put the body of this loop into a function so that date_list_with_notified_flag could be populated like this:
```
date_list_with_notified_flag = [{"date": date, "notified_completed": is_completed(date)} for date in dates_list]
```